forch/faucetizer.py

```diff
@@ -91,21 +91,22 @@ def process_devices_state(faucetizer: Faucetizer, devices_state: DevicesState):
 
 def load_faucet_config(file):
     """Load network state file"""
-    LOGGER.info('Loading faucet config file %s', file)
     with open(file) as config_file:
-        faucet_config = yaml.safe_load(config_file)
-    LOGGER.info('Loaded faucet config from %s', file)
-    return faucet_config
+        return yaml.safe_load(config_file)
 
 
-def write_faucet_config(faucet_config, out_path):
-    """Write faucet config to file"""
-    try:
-        with open(out_path, 'w') as out_file:
-            yaml.dump(faucet_config, out_file)
-        LOGGER.info('Config wrote to %s', out_path)
-    except Exception as error:
-        LOGGER.error('Cannot write faucet config: %s', error)
+def update_structural_config(faucetizer: Faucetizer, file):
+    """Read structural config from file and update in faucetizer"""
+    with open(file) as structural_config_file:
+        structural_config = yaml.safe_load(structural_config_file)
+        faucetizer.process_faucet_config(structural_config)
+
+
+def write_dynamic_config(faucetizer: Faucetizer, file):
```

Comment:
You're calling it _dynamic_ here but _behaviorial_ elsewhere -- likely good to be consistent.

Reply:
Done. Always using behavioral instead of dynamic now.